README.md

```diff
@@ -25,19 +25,29 @@ This will deploy hlf-k8s with:
 - 1 orderer `MyOrderer`
 - 2 organizations: `MyOrg1` and `MyOrg2`
 
-### Install a custom chaincode
+### Running a specific version
+
+To deploy locally a specific version of hlf-k8s, the recommended way is the following:
+```bash
+export SUBSTRA_HLF_VERSION=0.0.16
```

Comment:
The `export` is unnecessary here, unless I'm missing something?

```suggestion
SUBSTRA_HLF_VERSION=0.0.16
```

Reply:
It is necessary since without the export the env var would only live until the end of the statement, which is void here.
You can check this behavior with:
```sh
$ TEST_ENV=doesnotlivelongenough
$ env | grep longenough
$ # won't return anything, var is not in env
```

An alternative would be to repeat the env var before each statement, but that would be noisy:
```sh
$ TEST_ENV=doesnotlivelongenough env | grep longenough
TEST_ENV=doesnotlivelongenough
```

Besides, since the name is prefixed by `SUBSTRA_` it won't pollute the env (like colliding with an used-elsewhere value).

Reply:
It depends if you need to keep this var in env var or not, for a shell session, it looks like you don't need to export it no ?
```bash
TEST_ENV=doesnotlivelongenough
echo $TEST_ENV
doesnotlivelongenough
```

Reply:
You are right, we could skip that since we don't need the env var from *inside* skaffold now.
I'll update the PRs accordingly.